lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -672,15 +668,15 @@ bool WebsocketTlsTPM::connect() {
         return !local_data->is_connecting() && EConnectionState::INITIALIZE != local_data->get_state();
     });
 
-    EVLOG_info << "Connect finalized with state: " << (int)local_data->get_state() << " Timeouted: " << timeouted;
+    EVLOG_debug << "Connect finalized with state: " << (int)local_data->get_state() << " Timeouted: " << timeouted;
```

Comment:
This could be moved at line 675, and be kept as 'info' because it should not happen a lot of times.